### PR TITLE
[tools] Remove commit date on nightly build tar file name

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit removes commit date on nightly build tar file name.

Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

### CODE REVIEW

File: tools/release_tool/onert_version.sh [hunk 1]
```diff
@@ -27,11 +27,7 @@ show_version() {
   current_version=${version_line#"Version:"}
 
   if [ $nightly -eq 0 ]; then
-    # Get head commit's date
-    pushd $nnfw_root > /dev/null
-    date=$(git log -1 --format=%ad --date=format:%y%m%d)
     echo $current_version-nightly-$date
```
Comment (seanshpark) [inline tools/release_tool/onert_version.sh:30]: `$date` ??? --> `$current_version-nightly` ?

Comment (hseok-oh) [reply tools/release_tool/onert_version.sh:30]: `$current_version-nightly`

**Review by chunseoklee — APPROVED** [review]

**Comment by chunseoklee** [comment]
@hseok-oh This should wait for internal CI change ?

**Comment by hseok-oh** [comment]
@chunseoklee No. Internal CI will get and use file name by this change automatically.

**Review by seanshpark — APPROVED** [review]
LGTM

**Comment by hseok-oh** [comment]
@nnfw-bot test cross-aarch64-runtime-test-release

**Pull request merged**